League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -117,68 +138,171 @@ private void startCloningRepositories()
+            }
+        }
+
         private void startDownloadingClient()
         {
             if (_abortInitiated)
             {
                 OnAbortSuccessfully();
                 return;
             }
+            if (Directory.Exists(Path.Combine(installDirectoryText.Text, "League of Legends_UNPACKED")))
+            {
+                Directory.Move(Path.Combine(installDirectoryText.Text, "League of Legends_UNPACKED"), Path.Combine(installDirectoryText.Text, "League of Legends"));
+            }
+
+            //League_Sandbox_Client
+
+            if (Directory.Exists(Path.Combine(installDirectoryText.Text, "League of Legends")))
+            {
+                Directory.Move(Path.Combine(installDirectoryText.Text, "League of Legends"), Path.Combine(installDirectoryText.Text, "League_Sandbox_Client"));
+            }
+
+            if (Directory.Exists(Path.Combine(installDirectoryText.Text, "League_Sandbox_Client")))
+            {                
+                downloadingProgressLabel.Text = "✔️";
+                startUnzippingClient();
+                return;
+            }
+
 
-            downloadingProgressLabel.Text = "--";
+            // downloadingProgressLabel.Text = "--";
+            // ProtocolProviderFactory.RegisterProtocolHandler("http",
+            //typeof(HttpProtocolProvider));
+            // ProtocolProviderFactory.RegisterProtocolHandler("https",
+            //     typeof(HttpProtocolProvider));
 
-            ProtocolProviderFactory.RegisterProtocolHandler("http", typeof(HttpProtocolProvider));
-            var resourceLocation = ResourceLocation.FromURL("http://gamemakersgarage.com/League_Sandbox_Client.7z");
-            // local file path
-            var uri = new Uri("http://gamemakersgarage.com/League_Sandbox_Client.7z");
+            // var resourceLocation = ResourceLocation.FromURL("https://drive.google.com/u/0/uc?export=download&confirm=iupg&id=1vr6kGpDK1Hq3Loh8-2z7dlmXSCGKqY2Z");
+            // // local file path
+            // var uri = new Uri("https://drive.google.com/u/0/uc?export=download&confirm=iupg&id=1vr6kGpDK1Hq3Loh8-2z7dlmXSCGKqY2Z");
             var localFilePath = Path.Combine(installDirectoryText.Text, Client_Folder_Name + ".7z");
             if (File.Exists(localFilePath))
             {
                 File.Delete(localFilePath);
             }
+            Process.Start(installDirectoryText.Text);
+            Process.Start("https://drive.google.com/open?id=1vr6kGpDK1Hq3Loh8-2z7dlmXSCGKqY2Z");
 
-            System.Windows.Forms.Timer timer = new System.Windows.Forms.Timer();
-            timer.Interval = 100;
+            downloadingProgressLabel.Text = $"Please download .7z to: {installDirectoryText.Text} and name it: {Client_Folder_Name}.7z";
 
-            // register download ended event
-            DownloadManager.Instance.DownloadEnded += (_, _2) =>
+            Clipboard.SetText(localFilePath);
+
+            while (!File.Exists(localFilePath))
```

Comment:
There are certainly better ways. (I think https://docs.microsoft.com/en-us/dotnet/api/system.io.filesystemwatcher.created?view=netcore-3.1 should be better)

This is not crucial currently. (I'll open an issue upon merge)